tests/good_test_vectors.rs

```diff
@@ -141,7 +141,7 @@ fn read_all_values(reader: &mut Reader<RawBinaryReader<BufReader<File>>>) -> Ion
                 reader.step_out()?;
             }
             String => {
-                let _text = reader.map_string(|_s| ())?;
+                let _text = reader.read_string()?;
```

Comment:
🗺️ This test used the non-object-safe methods we plan to remove. I've switched it to use the safe-but-more-expensive equivalent methods for now.